addons/mail/static/src/components/message_reply/message_reply.scss

```diff
@@ -0,0 +1,45 @@
+.o_MessageReply {
+    display: block;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+    cursor: pointer;
+    padding-left: $o-mail-message-sidebar-width;
+
+    &:hover {
+        background-color: gray('200');
+    }
+}
+
+.o_MessageReply_author::before {
+    @include o-position-absolute($left: $o-mail-message-sidebar-width / 2, $bottom: 0);
+    content: "";
+    display: inline-block;
+    border-left: $border-width solid currentColor;
+    border-top: $border-width solid currentColor;
+    border-radius: $o-mail-rounded-rectangle-border-radius-sm 0 0 0;
+    height: 1em;
+    width: 2em;
+    opacity: .5;
+}
+
+.o_MessageReply_author::after {
+    content: ":";
+}
```

Comment:
No `content` in SCSS. This is relevant to UI, so should be explicit in template.
It looks like you want something like `X: Y`. To properly handle l10n, should use string interpolation on translatable text format, e.g. `_t("%s: %s")`, as not all local use ":" + whitespace in this context.